include/geos/operation/polygonize/PolygonizeGraph.h

```diff
@@ -152,7 +151,7 @@ class GEOS_DLL PolygonizeGraph: public planargraph::PlanarGraph {
 	 *
 	 */
 	void convertMaximalToMinimalEdgeRings(
-			std::vector<PolygonizeDirectedEdge*> &ringEdges);
+			std::vector<PolygonizeDirectedEdge*> ringEdges);
```

Comment:
Why copying the vector rather than passing a reference ?

Reply:
I havent finished, I need to add const to some parameters, so that it uses the move constructor of the vector